packages/manager/modules/vps/src/dashboard/rebuild/vps-rebuild.controller.js

```diff
@@ -1,21 +1,59 @@
 import get from 'lodash/get';
+import { AVAILABLE } from '../vps-dashboard.constants';
 
 export default class VpsRebuildController {
   /* @ngInject */
-  constructor(vpsRebuild) {
+  constructor(atInternet, vpsRebuild, $http) {
+    this.$http = $http;
     this.vpsRebuild = vpsRebuild;
+    this.atInternet = atInternet;
   }
 
   $onInit() {
+    this.osRtmCompatibility = false;
     this.vpsOptions = {
       doNotSendPassword: false,
+      installRTM: true,
     };
   }
 
+  onOsChange(modelValue) {
+    this.$http
+      .get(
+        `/vps/order/rule/osChoices?datacenter=${this.vps.location.datacentre}&os=${modelValue.name}`,
+      )
+      .then(({ data }) => {
+        this.osRtmCompatibility = data.choices[0].status === AVAILABLE;
```

Comment:
Always one choice ?

Reply:
in my test yes but you have right and un change it for a better binding